httpi.gemspec

```diff
@@ -19,11 +19,11 @@ Gem::Specification.new do |s|
   s.add_dependency 'rack'
   s.add_dependency 'socksify'
 
-  s.add_development_dependency 'rubyntlm', '~> 0.3.2'
-  s.add_development_dependency 'rake',     '~> 10.0'
-  s.add_development_dependency 'rspec',    '~> 2.14'
-  s.add_development_dependency 'mocha',    '~> 0.13'
-  s.add_development_dependency 'puma',     '~> 2.3.2'
+  s.add_development_dependency 'rubyntlm',    '~> 0.3.2'
+  s.add_development_dependency 'rake',        '~> 10.0'
+  s.add_development_dependency 'rspec',       '~> 3.4.0'
```

Comment:
Well, I was fiddling with gems and `rspec` got updated so I updated in the gemspec file as well but if this is too much of an issue I'll revert it to `~> 2.14`.